EIPS/eip-3569.md

```diff
@@ -0,0 +1,139 @@
+---
+eip: 3569
+title: Sealed NFT Metadata Standard
+author: Sean Papanikolas (@pizzarob)
+discussions-to: https://github.com/ethereum/EIPs/pull/3569
```

Comment:
How was this merged with a PR as discussion url? 😅

Reply:
Sorry, this is my mistake. I've created a FEM thread and a PR to update: https://github.com/ethereum/EIPs/pull/4221